snapcraft/internal/pluginhandler.py

```diff
@@ -644,7 +651,7 @@ def _migratable_filesets(fileset, srcdir):
 
 
 def _migrate_files(snap_files, snap_dirs, srcdir, dstdir, missing_ok=False,
-                   follow_symlinks=False):
+                   follow_symlinks=False, fixup_func=None):
```

Comment:
This doesn't make for a particularly intuitive function from its signature, but I guess there's little reason to walk `dstdir` again.